hooks/post_push

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
```

Comment:
I still think people are going to think this is a githook, could you add a line explicitly saying this is a dockerhub hook